.github/workflows/build-ios-release.yml

```diff
@@ -75,3 +75,9 @@ jobs:
           SLACK_URL: ${{ secrets.SLACK_URL }}
           MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
           MATCH_GIT_BASIC_AUTHORIZATION: ${{ secrets.MATCH_GIT_BASIC_AUTHORIZATION }}
+
+      - name: Save Static File
```

Comment:
All sourcemap files are saved here.
Allowing for upload to sentry.io or debugging via command line tools.